lms/djangoapps/ccx/tests/test_views.py

```diff
@@ -638,7 +637,7 @@ def setUpClass(cls):
                 metadata={'graded': True, 'format': 'Homework'})
             for _ in xrange(4)
         ]
-        problems = [
+        cls.problems = [
```

Comment:
Actually this is not used: I actually worked on these tests for another PR and I found really helpful to have these variable names, so I did not want to get rid of it even if it is not used.

I guess I can fins a better way to preserve this info